lua/starfall/libs_sv/entities.lua

```diff
@@ -783,7 +786,7 @@ function ents_methods:getAllConstrained(filter)
 			if not filter or filter.Wire then
 				if istable(ent.Inputs) then
 					for _, v in pairs(ent.Inputs) do
-						if isentity(v.Src) and v.Src:IsValid() then
+						if IsValid(v.Src) then
```

Comment:
Need to verify that passing tables or non-entity types doesn't error this function

Reply:
Should be fine if it errors